### PR TITLE
WebContent+WebDriver: Move all window commands from section 11.8 over to WebContent

### PR DESCRIPTION
**Pull request by trflynn89**
Just to show this stuff still works :wink: 

[Screencast from 2022-11-09 12-28-49.webm](https://user-images.githubusercontent.com/5600524/200899612-5c5dc7f6-2a44-466d-b890-4dfc9bb8fa23.webm)

### CODE REVIEW

File: Userland/Applications/Browser/Tab.cpp [hunk 1]
```diff
@@ -248,6 +248,22 @@ Tab::Tab(BrowserWindow& window)
         update_status({}, count_waiting);
     };
 
+    view().on_restore_window = [this]() {
+        this->window().show();
+        this->window().move_to_front();
```
Comment (linusg) [inline Userland/Applications/Browser/Tab.cpp:253]: omit `this`

Comment (linusg) [reply Userland/Applications/Browser/Tab.cpp:253]: same below

File: Userland/Applications/Browser/Tab.cpp [hunk 2]
```diff
@@ -264,6 +264,17 @@ Tab::Tab(BrowserWindow& window)
...
+    view().on_maximize_window = [this]() {
+        this->window().set_maximized(true);
+        return this->window().rect();
+    };
+
+    view().on_minimize_window = [this]() {
+        this->window().set_minimized(true);
+        m_web_content_view->set_system_visibility_state(false);
+        return this->window().rect();
+    };
```
Comment (linusg) [inline Userland/Applications/Browser/Tab.cpp:276]: Likewise

**Review by linusg — COMMENTED** [review]
Not entirely happy with the last commit, can these not go away in the same commit they're being moved over?

File: Userland/Applications/Browser/Tab.cpp (hunk 1 above)
Comment (trflynn89) [reply Userland/Applications/Browser/Tab.cpp:253]: Can't - the compiler couldn't disambiguate between the `window()` method and a local variable named `window` further up in the file, even though only `this` is captured.

**Comment by linusg** [comment]
Hmm, then maybe a small one getting rid of the previously removed ones and then as suggested? No big deal either way, just want to avoid having one of these at the end of every migration PR :)

**Review by linusg — APPROVED** [review]
:yakkiss:

**Pull request merged**